jf-common/jf-common-vendor-blobs.mk

```diff
@@ -152,4 +152,6 @@ PRODUCT_COPY_FILES += \
 	vendor/samsung/jf-common/proprietary/vendor/lib/libwvdrm_L1.so:system/vendor/lib/libwvdrm_L1.so \
 	vendor/samsung/jf-common/proprietary/vendor/lib/libwvm.so:system/vendor/lib/libwvm.so \
 	vendor/samsung/jf-common/proprietary/vendor/lib/libWVStreamControlAPI_L1.so:system/vendor/lib/libWVStreamControlAPI_L1.so \
-	vendor/samsung/jf-common/proprietary/vendor/lib/mediadrm/libwvdrmengine.so:system/vendor/lib/mediadrm/libwvdrmengine.so
+	vendor/samsung/jf-common/proprietary/vendor/lib/mediadrm/libwvdrmengine.so:system/vendor/lib/mediadrm/libwvdrmengine.so \
+	vendor/samsung/jf-common/proprietary/lib/libstlport.so:system/lib/libstlport.so \
+	vendor/samsung/jf-common/proprietary/lib/libgnustl_shared:system/lib/libgnustl_shared
```

Comment:
Missing the .so part here.

But doesn't jf use the adreno blobs from flo? If so, then this shouldn't be needed. I'm using flo blobs on my serrano, and definitely don't need this. Also, libstlport can be compiled from source.